src/common/util/cards.ts

```diff
@@ -373,8 +363,10 @@ export function cardsFromJson(json: string, callback: (card: w.CardInStore) => a
       id: generateId(),
       metadata: {
         ...card.metadata,
-        created: card.metadata.created || Date.now(),
-        updated: card.metadata.updated || Date.now()
+        source: cardSourceForCurrentUser(),
+        created: (card.metadata && card.metadata.created) || Date.now(),
+        updated: Date.now(),
+        importedFromJson: Date.now()
```

Comment:
What is this field for?

Reply:
Right now I'm just checking if it's there to not show JSON-imported cards in carousels. In the future we could also compare the import timestamp to update timestamp to tell if the card has been edited since being imported, which is why I made it a timestamp rather than just a boolean.